cfg/CFG.h

```diff
@@ -107,7 +107,11 @@ class CFG final {
      */
     int numLocalVariables() const;
 
-    core::FileRef file;
+    // Stores MethodDef::loc (MethodDef::declLoc).
```

Comment:
This comment reads to me like "Stores MethodDef::loc (well, actually, MethodDef::declLoc)".  The two are not really the same thing, so why mention both of them in this comment?

Reply:
sorry looks like in an edit i deleted "not the ..." from the parenthesis somehow